control

```diff
@@ -80,11 +80,16 @@ def get_bzobj(bzopts):
         logging.debug("Bugzilla url empty, exclusion filter disabled")
         return None
     try:
+        # Allow bugzilla module to be embedded into test directory
+        sys.path.insert(0, os.path.dirname(job.control))
         import bugzilla  # Keep confined to this function
     except ImportError:
         logging.warning("Bugzilla status exclusion filter configured "
                         "but bugzilla python module unavailable.")
         return None
+    # We were never here, you didn't see or hear anything.
```

Comment:
Hmmmm, good point, that would be cleaner that the conditional.  I'm not too worried about it in the case of an exception, b/c anything breaking at this level will be cleaned up by autotest.  Okay, I'll sneak a fix in along with another PR :smile:   Thanks for checking!